static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsQuery.tsx

```diff
@@ -35,7 +36,9 @@ export const useProjectRawWebVitalsQuery = ({transaction, tag, dataset}: Props =
       ],
       name: 'Web Vitals',
       query: [
-        'transaction.op:pageload',
+        // TODO: inp spans don't have a transaction.op.
+        // Plan to update this filter to also check span.op:ui.interaction.click once we have the ability.
+        'transaction.op:[pageload,""]',
```

Comment:
I should be able to update this query to `transaction.op:pageload OR span.op:ui.interaction.click` now, due to this pr: https://github.com/getsentry/sentry/pull/66423

I'll update this in a follow up pr